src/urllib3/util/ssl_.py

```diff
@@ -293,6 +294,12 @@ def create_urllib3_context(
         # We do our own verification, including fingerprints and alternative
         # hostnames. So disable it here
         context.check_hostname = False
+
+    # Enable logging of TLS session keys via defacto standard environment variable
+    # 'SSLKEYLOGFILE', if the feature is available (Python 3.8+).
+    if hasattr(context, "keylog_filename"):
```

Comment:
Can we also add a check that the environment variable is "truthy" before assigning? In case users do `SSLKEYLOGFILE=`.

Reply:
I don't think checking would add any value: the ssl code has to do input validation / graceful failure handling anyway, e.g. in case of non-existent or unwritable paths. It already ignores an empty value.